app/src/main/res/values/strings.xml

```diff
@@ -249,10 +249,10 @@
     <string name="failed_to_disable_op">Could not disable the requested app op</string>
     <string name="rules_not_applied">Rules are not applied</string>
     <string name="str_logo">Logo</string>
-    <string name="pref_global_blocking_enabled">Global component blocking</string>
+    <string name="pref_global_blocking_enabled">Block right away</string>
```

Comment:
This appears to be unsuitable for a title. Use more formal phrases such as “Instant component blocking”.